backend/src/nodes/nodes/image_filter/blur_bilateral.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageFilterCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, SliderInput
+from ...properties.outputs import ImageOutput
+from ...utils.utils import get_h_w_c
+
+
+@NodeFactory.register("chainner:image:bilateral_blur")
+class BlurNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply surface/bilateral blur to an image."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Diameter", controls_step=1, default=12),
+            SliderInput(
+                "Color Sigma",
+                controls_step=1,
+                default=25,
+                scale="log",
+                minimum=0,
+                maximum=1000,
+            ),
+            SliderInput(
+                "Space Sigma",
+                controls_step=1,
+                default=25,
+                scale="log",
+                minimum=0,
+                maximum=1000,
+            ),
+        ]
+        self.outputs = [ImageOutput(image_type="Input0")]
+        self.category = ImageFilterCategory
+        self.name = "Surface Blur"
+        self.icon = "MdBlurOn"
+        self.sub = "Blur"
+
+    def run(
+        self,
+        img: np.ndarray,
+        diameter: int,
+        sigma_color: int,
+        sigma_space: int,
+    ) -> np.ndarray:
+
+        if diameter == 0 or sigma_space == 0:
```

Comment:
The 0==1 thing applies to both sigmas. [More info](https://github.com/opencv/opencv/issues/22886#issuecomment-1331383727).

```suggestion
        if diameter == 0 or sigma_color == 0 or sigma_space == 0:
```